UofS-Pulse-Binfo/analyzedphenotypes
# Qualitative frequency chart

Documentation:

## Description
This makes for cleaner qualitative charts where a single germplasm may have different values depending across replicates (e.g. lodging). Previous charts ended up with 1/2 for the scale to show that some germplasm had mixed calls. This new version will not show that but on germplasm pages, two bars will be highlighted.

<img width="820" alt="Screen Shot 2019-05-23 at 3 10 05 PM" src="https://user-images.githubusercontent.com/1566301/58286994-37a55480-7d6d-11e9-9246-fcde1b40cdcc.png">


## Testing?




1. Go to the trait distribution chart
2. Select a qualitative trait.
3. Check that it displays a frequency of occurrence and that it makes sense.
